### ISSUE TITLE
Installed as Windows service the screenshot is always a black image instead of the actual screen

### ISSUE THREAD

**Issue by kwoxer** [bencevans/screenshot-desktop#208]
**Describe the bug**
Installed as a Windows service the screenshot is always a black image instead of the actual screen.

**To Reproduce 1**
1. Have a `server.js` with `screenshot-desktop`
2. Run `server.js` with `node-windows`
3. Now the screenshot is just a black image

**To Reproduce 2**
Steps to check when it works:
1. Have a `server.js` with `screenshot-desktop`
2. Run `server.js` without `node-windows` (in command line)
3. Now the screenshot is correct

**Expected behavior**
The image should ALWAYS contain the current screen.

**Environment (please complete the following information):**
 - Window Server 2019 (but same also on usual Windows 10)
 - Latest node packages

**Additional context**
This is the command I installed the screenshot server

```
var Service = require('node-windows').Service;
var svc = new Service({
  name: 'Screenshot tool',
  script: 'C:\\path\\server.js'
});
svc.on('install', function(){
  svc.start();
});
svc.install();
```

**Idea**

So far what I have seen that the issue is the session ID. So the session 0 is never a real user. So it always stays black. 

![image](https://user-images.githubusercontent.com/1167229/120501970-54e07500-c3c2-11eb-8bef-8f536485c484.png)

But maybe to set the resolution somewhere or changing the session ID? Is that possible maybe in this node package directly?

**Comment by bencevans** [comment]
Hi @kwoxer, thanks for the detailed report. I'm afraid I've just closed an issue covering the same at https://github.com/bencevans/screenshot-desktop/issues/185. A PR would be welcome if you find a way to solve it but shall close this as per the last comment on #185. If there's an option in the service setup to run as a particular user this may help but I believe when I tried it was still sandboxed away from the "real" desktop session.
